### PR TITLE
Debug webpack apps

### PR DESCRIPTION
**Pull request by offtherailz**
This pull request adds some information about how to debug apps using Webpack's dev server, with some hints for troubleshooting.

### CODE REVIEW

**Comment by auchenberg** [comment]
@offtherailz Not sure what the intent of this guide is. Is it to debug webpack itself, or apps bundled with webpack?

**Comment by offtherailz** [comment]
Debug apps bundled with webpack

File: README.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ A collection of recipes for using VS Code with particular technologies.
 
 - [Debugging Node.js with Nodemon](https://github.com/Microsoft/vscode-recipes/tree/master/nodemon)
 
+- [Debugging Webpack apps](https://github.com/Microsoft/vscode-recipes/tree/master/debugging-webpack)
```
Comment (auchenberg) [inline README.md]: I'd suggest that we rename this to "Webpack bundled applications"

File: debugging-webpack/README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# Debugging webpack application
```
Comment (auchenberg) [inline debugging-webpack/README.md]: Same goes here, as this guide really isn't about debugging webpack applications, but web apps that has been bundled via webpack, and this is a guide on how you configure the generated sourcemaps for webpack to be compatible with VS Code.

**Comment by auchenberg** [comment]
In general I think this content would probably be better as documentation than a debugging recipe. The valuable bits are in the last section. https://github.com/Microsoft/vscode-docs

**Comment by auchenberg** [comment]
Closing as this has stalled.

**Comment by offtherailz** [comment]
Sorry if i didn't reply for a long time. I'll probably check https://github.com/Microsoft/vscode-docs and propose this doc there.
Thanks for the support

**Pull request closed without merging**